server/api/roster/roster.types.ts

```diff
@@ -1,10 +1,10 @@
 import { CustomColumnConfig } from './custom-roster-column.model';
 
 export interface CustomColumns {
-  [key: string]: CustomColumnValue
+  [columnName: string]: RosterColumnValue | undefined
```

Comment:
This makes me nervous too. I always clinch when I see undefined on an object that can be serialized.

Is there a way to use Partial<CustomColumns> in the cases where we use this for import data gathering but generally disallow the `undefined` elsewhere?

Reply:
I think that was actually a remnant from before the `undefined` guard I put in the `setColumnValue()` function. We should be safe to just strip `undefined` out.